appveyor.yml

```diff
@@ -4,17 +4,13 @@ environment:
     # node.js
     - nodejs_version: "9.0"
     - nodejs_version: "8.0"
-    - nodejs_version: "7.0"
-    - nodejs_version: "6.0"
-    - nodejs_version: "5.0"
-    - nodejs_version: "4.0"
 
 # Install scripts. (runs after repo cloning)
 install:
   # Get the latest stable version of Node.js or io.js
   - ps: Install-Product node $env:nodejs_version
   # install modules
-  - npm install
+  - appveyor-retry npm install
```

Comment:
`appveyor-retry` is to ensure it install everything properly. Because pretty weirdly some times it fails. Or at least it was that way before year or more.